community-members.md

```diff
@@ -56,6 +56,12 @@ Repo: [open-telemetry/opentelemetry-java](https://github.com/open-telemetry/open
 
 The list of active members (both "approvers" and "maintainers") for the OpenTelemetry Java API and SDK can be found in the [open-telemetry/opentelemetry-java README file](https://github.com/open-telemetry/opentelemetry-java#owners).
 
+## Java Instrumentation
```

Comment:
nit: can we do a `###` here to suggest some hierarchy by language? (just trying to plan ahead)

```suggestion
### Java Instrumentation
```